scripts/validation/strip_first_csv_col.py

```diff
@@ -0,0 +1,43 @@
+#!/bin/env python3
+# SPDX-FileCopyrightText: Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import csv
+import io
+
+
+def parse_args():
+    argparser = argparse.ArgumentParser("Strips the first column in a csv file sending the results to stdout")
+    argparser.add_argument("input_file", help="File to strip the column from")
+    args = argparser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+    output = io.StringIO()
+    with open(args.input_file) as fh:
+        reader = csv.reader(fh)
+        for row in reader:
+            writer = csv.writer(output)
+            writer.writerow(row[1:])
+
+    print(output.getvalue().strip())
```

Comment:
Why do we need this file? Wouldnt it be easier to filter this column in Morpheus or add the column to the test data?

Reply:
The problem is that when we read from the topic we get a DataFrame that is (potentially) smaller than the source data, and since `WriteToFileStage` is always appending to the output file what we get is that the ID values restart back to 0 on subsequent messages.

When the topic has multiple partitions then we get this all the time and the resulting CSV file ends up looking like:
```
,v1,v2,v3,v4
0,0.1,0.7,0.7,0.7
1,0.5,0.5,0.7,0.9
2,1.0,0.2,0.2,0.9
3,0.9,0.0,0.1,0.5
4,0.0,0.5,0.5,0.6
5,0.2,0.5,0.1,0.3
6,0.5,1.0,0.4,0.7
0,1.0,0.9,0.5,0.9
1,0.5,0.8,0.6,0.0
2,0.3,0.4,0.1,0.4
3,0.1,0.3,1.0,0.6
4,0.5,0.3,0.6,0.8
5,0.1,0.9,0.1,0.3
0,0.0,0.5,0.6,0.3
1,0.5,0.6,0.2,0.9
2,1.0,0.1,0.1,0.3
3,0.6,0.0,0.5,0.5
4,0.0,0.3,0.5,0.6
5,0.6,0.8,0.8,0.1
6,0.8,0.8,1.0,0.6
```

Reply:
Looks like I can add a flag to the `WriteToFileStage` to exclude the ID column, as it looks like cudf won't let us exclude it in the serialize stage.